CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## IN PROGRESS
 
+## [8.0.0](https://github.com/folio-org/ui-tenant-settings/tree/v8.0.0)(2023-10-12)
+
```

Comment:
Could you please add 
`[Full Changelog](https://github.com/folio-org/ui-tenant-settings/compare/v7.4.0...v8.0.0)` ?

Reply:
done!